### DISCUSSION TITLE
No output web page

### DISCUSSION THREAD

**Discussion by ElleWilte** [community/community#31514] [Other Feature Feedback, Questions, & Ideas]
The web page https://github.com/ElleWilte/huiswerk_les9.git/Fort Rammekens generates the output '404 This is not the web page you are looking for'. Please tell me why.

**Comment by airtower-luna** [comment]
I assume you mean the deployed page at https://ellewilte.github.io/huiswerk_les9/?

In that case the explanation is that there is no `index.html` file in your repository, which is what opens if you access the site without a file name. If you access the file you have it works: https://ellewilte.github.io/huiswerk_les9/Fort%20Rammekens.html

**Comment by dikkyryan** [comment]
there is no index.html file in your repository
